caf/caf/cli_commands.py

```diff
@@ -310,10 +310,13 @@ def delete_tag(**kwargs) -> int:
     except RepositoryNotFoundError:
         _print_error(f'No repository found at {repo.repo_path()}')
         return -1
-    except RepositoryError as e:
-        _print_error(f'Repository error: {e}')
+    except TagNotInTagsDirError as e:
+        _print_error(f'Tag error: {e}')
         return -1
-    
+    except ValueError as e:
```

Comment:
This is not supposed to happen and throwing a ValueError in the user's face is unprofessional, even if you give it a custom message. Why do you need to catch such a broad exception type here? If it gets here, this is a bug that the programmer needs to see and fix, so it should be as explosive as possible. Remove this except branch.